README.md

```diff
@@ -26,7 +26,12 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose person
+  puts "Would you take this rose and help out an old begger, #{person}"
+end
+
+offerRose "young prince"
+
 ```
```

Comment:
:+1: A little bit of a trick question here -- correct, but remember that Rubyists tend to prefer snake_case